lambdas/services/bulk_upload_metadata_processor_service.py

```diff
@@ -244,7 +244,8 @@ def validate_and_correct_filename(self, file_metadata: MetadataFile) -> str:
             return file_metadata.file_path
         except LGInvalidFilesException:
             return self.metadata_formatter_service.validate_record_filename(
```

Comment:
This method is also called without nhs number on like 264. Is that cool?

Reply:
I will test this to make sure, for some reason that means of ingestion doesn't require it.